docs/ppp/vision-21112022.md

```diff
@@ -0,0 +1,85 @@
+
+That is it, it is that dead simple: if you keep your Bitcoin on an exchange instead of putting them into self custody, you are going to have a bad time.
+Remember MtGox?
+
+> "That's too long ago, exchanges got better", John Doe, holds his Bitcoin on FTX.
+
+John, we are sorry to tell you: "Not your keys, not your coins!".
+Did you learn your lesson?
+
+> "But, but, I wanted to trade!", John Doe
+
+That is a common response we get, and we hear you!
+
+This is why we are bringing noncustodial peer-to-peer financial products to Lightning.
+
+![2022-11-21-goosbumps.png](../assets/2022-11-21-goosbumps.png)
+
+## The Vision
+
+Imagine a world just days before [hyperbitcoinization](https://bitcoinmagazine.com/hyperbitcoinization) finally hits.
+You walk along the beach enjoying the warmth of the sunset.
+
+You stop at a small food truck selling you a juicy steak, medium rare without sides, just perfect.
+You tap with your phone to pay with Lightning and sit down to enjoy the orange light of the sunset.
+
+Someone sells you a Tequila Sunrise - ahm - Tequila Sunset, freshly prepared right in front of you.
+
+While you take a sip from the drink, you read the latest news of the fed raising interest rates while throwing on the money printer at the same time to fight inflation.
+You casually take out your phone and open a long position on Bitcoin, knowing, that the last actions of the government will push the price even higher.
+
+Relaxed you lay back and open a streaming service which charges per seconds paid via Lightning.
+You wanted to check in on your favorite team, El Salvadore, who very likely will become world champion this year.
+Watching the team in orange shirts pushes your adrenaline higher and makes you euphoric.
+You bet that El Salvadore will win and are happy to put a few Sats on that.
+
+An entertaining game later, you stack the profits of your bet and wander home.
+
+At home, you move the Sats from today's winnings into the noncustodial savings account which eventually will pay for your son's tuition.
+You check in on your long position seeing it is going well and double down once more before going to bed.
+
+Knowing all you did today was secured by Bitcoin, fully noncustodial and without having to trust anyone, you fall asleep with a smile on your face.
+
+Sounds to good to be true? Let us introduce you **10101 the One-Stop Bitcoin Wallet for the sovereign Bitcoiner**.
+
+## 10101
+
+![10101.png](../assets/2022-11021-10101.png)
+
+10101 is a **noncustodial** on-chain and Lightning **wallet** for the masses.
+It has all the bells and whistles a good Bitcoin wallet needs with an extra level of pizazz.
+
+1. You can pay **on-chain** or **off-chain** to anyone on the Lightning Network, and you can receive on-chain payments or payments from anyone on the Lightning Network.
+2. It will allow you to **trade** CFDs, Options and other derivatives secured by Bitcoin.
+3. You can **bet** on who becomes the world champion at Fifa22, and
```

Comment:
Since I don't believe we're gonna have sports betting by December 😛 

```suggestion
3. You can **bet** on who will be crowned champion at the FIFA 2026 World Cup.
```

Removed the and because I think you need to use semicolons throughout the whole list for it to make it sense. Full-stops are fine imo.

Reply:
haha, happy to change that :D